### ISSUE TITLE
make sure libjeffpc works on Linux

### ISSUE THREAD

**Issue by jeffpc** [jeffpc/nx01#70]
It was developed on Illumos and it's been a while since the last time it was compiled on Linux.  It will be significantly easier to test once #63 is done.

**Comment by jeffpc** [comment]
It looks like the only problem is the use of illumos-style mapfile.  That can be easily worked around by disabling the mapfile use in the libjeffpc CMakeLists.txt:

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index fe6d27c..c656bc4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -98,7 +98,7 @@ target_link_libraries(jeffpc
    uuid
 )

-target_apply_mapfile(jeffpc mapfile-vers)
+#target_apply_mapfile(jeffpc mapfile-vers)

 install(TARGETS    jeffpc DESTINATION ${CMAKE_INSTALL_LIBDIR}
    PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ)
```

**Comment by jeffpc** [comment]
Beware of #83

**Comment by jeffpc** [comment]
Fixed as of libjeffpc commit 47dd6d02a07c1be54981d967906db1469903e364.
